src/Api/Controllers/OrganizationsController.cs

```diff
@@ -319,10 +323,21 @@ public async Task PostSubscription(string id, [FromBody] OrganizationSubscriptio
         {
             throw new NotFoundException();
         }
-
         await _organizationService.UpdateSubscription(orgIdGuid, model.SeatAdjustment, model.MaxAutoscaleSeats);
     }
 
+    [HttpPost("{id:guid}/sm-subscription")]
+    [SelfHosted(NotSelfHostedOnly = true)]
+    public async Task PostSmSubscription(Guid id, [FromBody] OrganizationSmSubscriptionUpdateRequestModel model)
```

Comment:
@r-tome I see this was your suggestion, any comment?

Reply:
It is not _necessary_ because the parameter type is already `Guid` but it can be useful in the future if we want to generate some documentation or use Swagger.

Reply:
resolved